desk/sur/diary.hoon

```diff
@@ -120,14 +120,24 @@
   $%  [%block p=block]
       [%inline p=(list inline)]
   ==
+::  $listing: recursive type for infinitely nested <ul> or <ol>
++$  listing
+  $%  [%list p=?(%ordered %unordered) q=(list listing) r=(list inline)]
+      [%item p=(list inline)]
+  ==
 ::  $block: post content that sits outside of the normal text
 ::
 ::    %image: a visual, we record dimensions for better rendering
 ::    %cite: an Urbit reference
+::    %header: a traditional HTML heading, h1-h6
+::    %listing: a traditional HTML list, ul and ol
 ::
 +$  block
   $%  [%image src=cord height=@ud width=@ud alt=cord]
       [%cite =cite:c]
+      [%header p=?(%h1 %h2 %h3 %h4 %h5 %h6) q=inline]
```

Comment:
should be `q=(list inline)`

Reply:
bah ok fine

Reply:
come on..., it's like a 10 line change tops ;)